main.go

```diff
@@ -40,17 +42,57 @@ const assetPath = "web"
 var embeddedFiles embed.FS
 
 func main() {
-	e := newEchoServer()
+	// first load the config
+	config := config.LoadConfig()
+	config.Print(log.Writer())
+
+	// then initialize the data storage and fill it with the initial transactions
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	store := data.NewStore()
+	loadHistory(store, config)
```

Comment:
app has to be restarted to show new data?

Reply:
As shown in the PR description, wip: `Then connect Nats`